Lib/test/test_math.py

```diff
@@ -809,8 +809,8 @@ def testHypot(self):
         # Test allowable types (those with __float__)
         self.assertEqual(hypot(12.0, 5.0), 13.0)
         self.assertEqual(hypot(12, 5), 13)
-        self.assertEqual(hypot(1, -1), math.sqrt(2))
-        self.assertEqual(hypot(1, FloatLike(-1.)), math.sqrt(2))
+        self.assertEqual(hypot(-1), 1.0)
+        self.assertEqual(hypot(FloatLike(-1.)), 1.0)
```

Comment:
All other tests here are 2-argument. And `hypot()` initially only supported 2 arguments, so there may be a special case for 2 arguments. Would testing that `hypot(0.75, -1) == 1.25` help?

If you test argument conversion here, it may be worth to test this also for other argument: `hypot(-1, 0.75)`

Reply:
> All other tests here are 2-argument. 

Not all, see below e.g. for 0's/inf/nan.

> Would testing that hypot(0.75, -1) == 1.25 help?

Yeah, will work too, just little longer.  I'll add also tests with reversed arguments.